core/src/main/scala/pureconfig/error/ConfigReaderFailure.scala

```diff
@@ -298,6 +299,16 @@ final case class NoValidCoproductChoiceFound(value: ConfigValue, location: Optio
 }
 
 /**
+ * A failure representing the inability to read a requested file.
+ */
+final case class CannotReadFile(path: Path) extends ConfigReaderFailure {
+  val location = None
+
+  def description = s"Unable to read file: ${path.toString}"
```

Comment:
The addition of `CannotReadFile` was already in #245 and there was a discussion [there](https://github.com/pureconfig/pureconfig/pull/245#discussion_r118805343) about this topic. I agree with you, but I think it's a different issue and PR ;)